src/App.tsx

```diff
@@ -1,9 +1,24 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [searchedMovie, setSearchedMovie] = useState('');
+
+  const handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setSearchedMovie(event.currentTarget.value);
+  };
+
+  function isIncludes(movie: string) {
```

Comment:
1. You should use more understandable name for function, it must show what function actually do (what is includes and where);
2. I think it is better to rename parameter (not movie, probably movieData)